manticore/core/workspace.py

```diff
@@ -31,7 +31,26 @@ def manager():
     return _manager
 
 
-class Store(object):
+class Testcase:
+    def __init__(self, workspace, prefix):
+        self._num = workspace._increment_id()
+        self._prefix = prefix
+        self._ws = workspace
+
+    @property
+    def prefix(self):
```

Comment:
I added a `prefix` property here which is used/passed to output's `. save_testcase` for generating native testcases: https://github.com/trailofbits/manticore/pull/1305/files#diff-92b1b59d64fb129749dcf0ede70b5131R39

This `save_testcase` should probably be abstracted out.